draft-ietf-taps-interface.md

```diff
@@ -652,7 +652,9 @@ make a peer-to-peer `Rendezvous`.
 
 If more than one Remote Endpoint is specified on the Preconnection, then
 all the Remote Endpoints on the Preconnection SHOULD represent the same
```

Comment:
```suggestion
all the Remote Endpoints on the Preconnection should represent the same
```
AS per this suggestion, I now think that the solution is to de-capitalise the SHOULD ... it's not something that the local Transport Services system has any control over?!   The "to the extent ..." thing is okay, I think it doesn't really change anything. It's the SHOULD that was indeed confusing. On the server side, I'd say MUST. On the client side, one doesn't have control.